erp/export/base-schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://specs.frictionlessdata.io/schemas/table-schema.json",
+  "name": "acceslibre",
+  "title": "Schéma des données d'accessibilité des ERPs",
+  "description": "Spécification du fichier d'échange relatif aux données d'accessibilité des Établissements Recevant du Public (ERP).",
+  "keywords": [
+    "accessibilite",
+    "access",
+    "a11y"
+  ],
+  "countryCode": "FR",
+  "homepage": "https://github.com/MTES-MCT/acceslibre-schema",
+  "path": "https://github.com/MTES-MCT/acceslibre-schema/raw/v0.0.1/schema.json",
+  "licenses": [
+    {
+      "title": "Etalab Licence Ouverte 2.0",
+      "name": "etalab-2.0",
+      "path": "https://spdx.org/licenses/etalab-2.0.html"
```

Comment:
This is still an open question, though I suspect we should only go with the most permissive license. Let's just not forget to obtain a decision made by the product team.